tests/PersistentSessionTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace atk4\data\tests;
+
+use atk4\core\AtkPhpunit;
+use atk4\data\Model;
+use atk4\data\Persistence;
+
+/**
+ * @coversDefaultClass \atk4\data\Persistence\Session
+ */
+class PersistentSessionTest extends AtkPhpunit\TestCase
+{
+    /**
+     * Tests.
+     */
+    public function testSession()
+    {
+        $a = [
+            'user' => [
+                1 => ['name' => 'John'],
+                2 => ['name' => 'Sarah'],
+            ],
+        ];
+
+        $p = new Persistence\Session($a);
```

Comment:
not sure how this works - phpunit does not initialize session by default. Will probably fail in practice.

Reply:
but strangely it didn't fail :)
Anyway I got that - this class is not as useful as I imagined. To much confusion and cons.